docs/developer/meta/ci/validation.md

```diff
@@ -0,0 +1,128 @@
+# Validation
+
+-----
+
+Various validations are [ran](https://github.com/DataDog/integrations-core/blob/master/.github/workflows/validate.yml) to check for correctness. There is a [reusable workflow](https://github.com/DataDog/integrations-core/blob/master/.github/workflows/run-validations.yml) that repositories may call with input parameters defining which validations to use, with each input parameter corresponding to a subcommand under the `ddev validate` command group.
+
+## Agent requirements
+
+```
+ddev validate agent-reqs
+```
+
+This validates that each integration version is in sync with the [`requirements-agent-release.txt`](https://github.com/DataDog/integrations-core/blob/master/requirements-agent-release.txt) file. It is uncommon for this to fail because the release process is automated.
+
+## CI configuration
+
+```
+ddev validate ci
+```
+
+This validates that all CI entries for integrations are valid. This includes checking if the integration has the correct [Codecov config](https://github.com/DataDog/integrations-core/blob/master/.codecov.yml), and has a valid [CI entry](testing.md#target-enumeration) if it is testable.
+
+!!! tip
+    Run `ddev validate ci --sync` to resolve most errors.
```

Comment:
```suggestion
    Run `ddev validate ci --fix` to resolve most errors.
```

Reply:
It's a bit confusing because we have `sync` on some commands and `fix` on some others

Reply:
nvm https://github.com/DataDog/integrations-core/pull/14421#discussion_r1172151321